core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/save/withSave.ts

```diff
@@ -0,0 +1,79 @@
+import { tapResponse } from '@ngrx/component-store';
+import { patchState, signalStoreFeature, type, withMethods } from '@ngrx/signals';
+import { rxMethod } from '@ngrx/signals/rxjs-interop';
+import { EMPTY, pipe } from 'rxjs';
+
+import { inject } from '@angular/core';
+
+import { catchError, switchMap, tap } from 'rxjs/operators';
+
+import { DotPageApiResponse, DotPageApiService } from '../../../../services/dot-page-api.service';
+import { UVE_STATUS } from '../../../../shared/enums';
+import { PageContainer } from '../../../../shared/models';
+import { UVEState } from '../../../models';
+
+/**
+ * Add methods to save the page
+ *
+ * @export
+ * @return {*}
+ */
+export function withSave() {
```

Comment:
Why this is a separate `signalStoreFeature`? It looks like a core method for the `withEditor`

Reply:
Is a separate file to not have a huge file